apps/code-examples/src/app/code-examples/indicators/key-info/basic/key-info-demo.component.spec.ts

```diff
@@ -0,0 +1,54 @@
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { TestBed } from '@angular/core/testing';
+import { SkyKeyInfoHarness } from '@skyux/indicators/testing';
+
+import { KeyInfoDemoComponent } from './key-info-demo.component';
+import { KeyInfoDemoModule } from './key-info-demo.module';
+
+describe('Basic key info', () => {
+  async function setupTest(options?: { layout?: 'vertical' | 'horizontal' }) {
```

Comment:
Can we type this with the new `SkyKeyInfoLayoutType`?

Reply:
after updating the tests, the `SkyKeyInfoLayoutType` is no longer exposed to the consumer